docs/faq/overview.md

```diff
@@ -0,0 +1,77 @@
+# Frequently Asked Questions (FAQ)
+
+## What is the Hedgehog Fabric?
+
+The Hedgehog Fabric is a topology of routers arranged in a spine-leaf architecture. A spine-leaf architecture is a type of [Clos network topology](https://en.wikipedia.org/wiki/Clos_network). In a spine-leaf architecture, the leaves are usually placed in racks and connected directly to the servers, whereas spines are connected only to leaves. In a spine-leaf architecture, the fundamental unit of connection is a layer 3 route.
+
+The Hedgehog Fabric is managed via Kubernetes objects and custom resource definitions.
+
+## What are the advantages of a spine-leaf architecture?
+
+A spine-leaf architecture is designed to facilitate traffic that is passing between servers inside of a data center, other architectures like core-access-aggregation are setup to facilitate traffic moving in and out of the data center. A spine-leaf architecture provides multiple paths between nodes which allows for router maintenance and resilience in the case of failures. The spine-leaf architecture allows allows for multiple points of egress via border leaf nodes. In a spine-leaf architecture the unit of connection is a layer 3 route. There are robust tools, queueing algorithms and hardware available to manage network traffic at layer 3. To manage the distribution of routes to switches inside the fabric a protocol such as  BGP, OSPF, or IS-IS is used. 
+
+### Spine Leaf Architecture Diagram
+```mermaid
+graph TD
+    S1([Spine 1])
+    S2([Spine 2])
+    S3([Spine 3])
+    L1([Leaf 1])
+    L2([Leaf 2])
+    L3([Leaf 3])
+    L4([Leaf 4])
+    WS1[[Worload Servers]]
+    WS2[[Worload Servers]]
+    WS3[[Worload Servers]]
+    WS4[[Worload Servers]]
+
+    S1 & S2 & S3 ---- L1 & L2 & L3 & L4 
+    L1 ---- WS1
+    L2 ---- WS2
+    L3 ---- WS3
+    L4 ---- WS4
+
+```
+In the above diagram Leaf and Spine routers, servers inside of a virtual private cloud can be attached to any leaf. In order for the servers to communicate routes are applied to leaf nodes and traffic that needs to pass from leaf 1 to leaf 2 could travel via any spine. The leaf uses [ECMP](https://en.wikipedia.org/wiki/Equal-cost_multi-path_routing) to decide which spine to use. An [EVPN](https://en.wikipedia.org/wiki/Ethernet_VPN) technology ensures that servers inside of a VPC are reachable at layer2 regardless of which leaf they are attached to the fabric. 
```

Comment:
```suggestion
The above diagram contains a Leaf and Spine routers. Servers inside of a virtual private cloud can be attached to any leaf. In order for the servers to communicate routes are applied to leaf nodes and traffic that needs to pass from leaf 1 to leaf 2 could travel via any spine. The leaf uses [ECMP](https://en.wikipedia.org/wiki/Equal-cost_multi-path_routing) to decide which spine to use. An [EVPN](https://en.wikipedia.org/wiki/Ethernet_VPN) technology ensures that servers inside of a VPC are reachable at layer 2 regardless of which leaf they are attached to in the fabric. 
```

Nit: I'd move the description above the diagram, so that readers know that they look at before they have to read the paragraph under it.